### ISSUE TITLE
Get battery Status brickpi with ev3dev

### ISSUE THREAD

**Issue by Negociation** [ev3dev/ev3dev#1179]
Hi , i need to check if my bricki will turn off , someone have any idea about how i do that ? 
I read about some people using the current voltage to know if brickpi will turn off...

I'm using EV3DEV with Python...

**Comment by dlech** [comment]
So you are saying you want to know how to get the battery voltage using python?

cc: @ev3dev/python

**Comment by Negociation** [comment]
Yeah , i want to know if can i get the voltage to check if my brickpi will turn off , and save my dataset before...

**Comment by WasabiFan** [comment]
https://python-ev3dev.readthedocs.io/en/ev3dev-stretch/other.html#power-supply
